analyze/index.ts

```diff
@@ -30,12 +30,31 @@ export default createBuilder<any>(
       const result = await build.result;
 
       if (result.success) {
-        const file = fs.readdirSync(builderConfig.outputPath).filter(f => f.includes('main-es2015'));
-        const mainFile = file.find(f => f.endsWith('.js'));
-        console.log(mainFile);
+        const file = fs.readdirSync(builderConfig.outputPath).filter(f => f.includes('es2015'));
+        const filesToRemove = file.filter(f => f.includes('polyfills') || f.includes('runtime'));
+        let filesToShow = file.filter(f => !filesToRemove.includes(f) && f.endsWith('.js'));
 
-        const explorerCommand = `source-map-explorer ${builderConfig.outputPath}/${mainFile}`;
+        let mainFile = filesToShow[0];
```

Comment:
I think sequence is not guaranteed we can use the below code to find the file which starts with main
filesToShow.find(file=> file.includes('main-es2015'));

Reply:
I am already filtering out the files to display using these lines.

``` typescript
    const file = fs.readdirSync(builderConfig.outputPath).filter(f => f.includes('es2015'));
    const filesToRemove = file.filter(f => f.includes('polyfills') || f.includes('runtime'));
    let filesToShow = file.filter(f => !filesToRemove.includes(f) && f.endsWith('.js'));
```

The `let mainFile = filesToShow[0];` statement will get the first item from the array, if the array has only one file then that will be executed by the analyzer. However, if the array has more than one files, then the if statement will be used to show the list of options and re-assign the file name selected by the user